src/agent.rs

```diff
@@ -376,7 +457,7 @@ impl<AGN: Agent> Bridge<AGN> for PrivateBridge<AGN> {
         // TODO Important! Implement.
         // Use a queue to collect a messages if an instance is not ready
         // and send them to an agent when it will reported readiness.
-        let msg = ToWorker::ProcessInput(SINGLETON_ID, msg).pack();
```

Comment:
This is a questionable change, and it might make sense to just change it to `Some(SINGLETON_ID)`.

Reply:
`SINGLETON_ID` seems to be intended to enforce that there is only one JobBridge or PrivateBridge. `Some(SINGLETON_ID)` is better